sdk/keyvault/test-resources.json

```diff
@@ -66,16 +59,25 @@
         "description": "The base time to add 2 hours to for SAS token expiration. The default is the current time."
       }
     },
-    "attestationUri": {
+    "attestationImageName": {
       "type": "string",
-      "defaultValue": "https://skrattestation.azurewebsites.net/",
+      "defaultValue": "keyvault-mock-attestation",
       "metadata": {
-        "description": "Test attestation service for Secure Key Release."
+        "description": "The name of the secure key release attestation image to deploy. Only deployed when enableHsm is true."
+      }
+    },
+    "attestationImageTag": {
+      "type": "string",
+      "defaultValue": "latest",
+      "metadata": {
+        "description": "The tag of the secure key release attestation image to use. The default is 'latest'. Only deployed when enableHsm is true."
       }
     }
   },
   "variables": {
     "kvApiVersion": "2019-09-01",
+    "attestationImageName":  "[concat('DOCKER|azsdkengsys.azurecr.io/', parameters('attestationImageName'), ':', parameters('attestationImageTag'))]",
```

Comment:
I'm going to rename this, it's a little confusing to have both a param and a variable using the same name